src/benchmarklib/synthetic_table_generator.cpp

```diff
@@ -210,4 +210,23 @@ std::shared_ptr<Table> SyntheticTableGenerator::generate_table(
   return table;
 }
 
+std::ostream& operator<<(std::ostream& stream, const ColumnDataDistribution column_data_distribution) {
+  switch (column_data_distribution.distribution_type) {
+    case DataDistributionType::Uniform:
+      stream << "Uniform"
+             << "_" << column_data_distribution.min_value << "_" << column_data_distribution.max_value;
+      break;
+    case DataDistributionType::NormalSkewed:
+      stream << "Skewed"
+             << "_" << column_data_distribution.skew_location << "_" << column_data_distribution.skew_scale << "_"
+             << column_data_distribution.skew_shape;
+      break;
+    case DataDistributionType::Pareto:
+      stream << "Pareto"
+             << "_" << column_data_distribution.pareto_scale << "_" << column_data_distribution.pareto_shape;
```

Comment:
Good point. I thought about it too. At the moment, I use the the DataDistributionType as part of the generated column name, which is why I avoid using brackets. Are brackets allowed as part of a column name?

Reply:
Do you use it in multiple places? If not, better avoid defining such "strange" output names globally and use your "string coding" only within your functions.

Reply:
I use it for column name generation only. I decided to remove the DataDistributionType in the column name completely, because we do not use it anywhere at the moment. To ensure uniqueness of column names, I added a simple counter.

A changed the related code in `calibration_table_generator.cpp` to:
```cpp
auto column_count = 0; // Needed to make column names unique
for (const auto& column_data_distribution : config->column_data_distributions) {
  std::stringstream column_name_stringstream;
  column_name_stringstream << data_type << "_" << encoding_type << "_" << column_count;

  auto column_name = column_name_stringstream.str();

  _column_data_distributions.emplace_back(column_data_distribution);
  _column_specs.emplace_back(
  ColumnSpecification(column_data_distribution, data_type, encoding_type, column_name));

  column_count++;
}
```

Therefore, we do not need this change anymore. If you feel like, it might be useful to have the data distribution type encoded in the column name, leave a comment, please.